PurchasesTests/Purchasing/StoreKitWrapperTests.swift

```diff
@@ -174,6 +174,7 @@ class StoreKitWrapperTests: XCTestCase, RCStoreKitWrapperDelegate {
     }
 
     func testDidRevokeEntitlementsForProductIdentifiersCallsDelegateWithRightArguments() {
+        #if swift(>=5.3)
```

Comment:
can't use the same check on objective-c and swift, because `swift(>=` doesn't work in obj-c and `__IPHONE_14_0` doesn't work in swift.